prog/dftb+/prg_dftb/initprogram.F90

```diff
@@ -1708,6 +1719,21 @@ subroutine initProgramVariables(input)
       call init(pMDIntegrator, pVelocityVerlet)
     end if
 
+    !c3091013
+    ! Initialising plumed
+    tPlumed = input%ctrl%tPlumed
+    if (tPlumed) then
+      call plumed_f_gcreate()
+      call plumed_f_gcmd("setNatoms"//char(0),nAtom)
+      call plumed_f_gcmd("setPlumedDat"//char(0),"plumed.dat"//char(0))
+      call plumed_f_gcmd("setNoVirial"//char(0))
+      call plumed_f_gcmd("setTimestep"//char(0),deltaT)
+      call plumed_f_gcmd("setMDEnergyUnits"//char(0),KJM_TO_H)
+      call plumed_f_gcmd("setMDLengthUnits"//char(0),NM_TO_B)
+      call plumed_f_gcmd("setMDTimeUnits"//char(0),PS_TO_AU)
+      call plumed_f_gcmd("init"//char(0),0)
+    end if
+
```

Comment:
We need interface definitions for the subroutines `plumed_f_gcreate()` and `plumed_f_gcmd()` to make sure, they are called correctly. They should be placed into a module in `lib_ext/plumed.F90`